content/articles/build-autocorrect-feauture-using-nlp-with-python/index.md

```diff
@@ -0,0 +1,285 @@
+---
+layout: engineering-education
+status: draft
+published: true
+url: /build-autocorrect-feature-using-nlp-with-python/
+title: Build an Autocorrect Feature using NLP with Python.
+description: The objective of this tutorial is to help the reader understand how Natural Language processing can be used with Python in building autocorrect features in systems. 
+author: antony-lia
+date: 2021-11-13T00:00:00-2:30
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/build-autocorrect-feature-using-nlp-with-python/hero.jpg
+    alt: Natural Language processing example image
+---
+One application of Natural Language Processing (NLP) is the Autocorrect function. This feature works on every smartphone keyboard regardless of the brand, it is specially programmed to generalize all the correct words that are in the vocabulary and tries to find the most similar words to those words, not in the vocabulary.
```

Comment:
remove the comma between brand and it. Replace "it is specially programmed to generalize all the correct words that are in the vocabulary and tries to find the most similar words to those words, not in the vocabulary." with " It is specially programmed to generalize all the correct words in the vocabulary and try to find the most similar words to those words, not in the vocabulary.	"